cas-server-core-configuration/src/main/java/org/apereo/cas/configuration/model/support/jpa/AbstractJpaProperties.java

```diff
@@ -0,0 +1,201 @@
+package org.apereo.cas.configuration.model.support.jpa;
+
+/**
+ * Common properties for all jpa configs.
+ *
+ * @author Dmitriy Kopylenko
+ * @since 5.0.0
+ */
+public abstract class AbstractJpaProperties {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [This abstract class does not have any abstract methods](https://www.codacy.com/app/mmoayyed/cas/file/2770326218/issues/source?bid=3256115&fileBranchId=3357337#l9)